api/main.go

```diff
@@ -20,3 +24,16 @@ func RootHandler(ctx echo.Context) error {
 	}
 	return ctx.JSON(http.StatusOK, resp)
 }
+
+func AppsHandler(ctx echo.Context) error{
+	db := cron.InitDB()
```

Comment:
Eventually, it might be useful to establish a single database connection pool and use it in all of the handlers. The `permissions` service does that by using functions that return other functions. See https://github.com/cyverse-de/permissions/blob/master/restapi/configure_permissions.go for more information. There's no rush on this, though; it's just something that would be helpful to get in place before we deploy the new service.